salt/metalk8s/addons/prometheus-operator/config/grafana.yaml.j2

```diff
@@ -0,0 +1,44 @@
+#!jinja|yaml
+
+{%- if pillar.addons.dex.enabled %}
+  {%- set dex_defaults = salt.slsutil.renderer('salt://metalk8s/addons/dex/config/dex.yaml.j2', saltenv=saltenv) %}
+  {%- set dex = salt.metalk8s_service_configuration.get_service_conf('metalk8s-auth', 'metalk8s-dex-config', dex_defaults) %}
+{%- endif %}
+{%- set control_plane_ingress_endpoint = salt.metalk8s_network.get_control_plane_ingress_endpoint() %}
+
+# Configuration of the Grafana service
+apiVersion: addons.metalk8s.scality.com
+kind: GrafanaConfig
+spec:
+  # Configure the Grafana Deployment
+  deployment:
+    replicas: 1
+  config:
+    grafana.ini:
+      analytics:
+        check_for_updates: false
+        reporting_enabled: false
+      paths:
+        data: /var/lib/grafana/
+        logs: /var/log/grafana
+        plugins: /var/lib/grafana/plugins
+        provisioning: /etc/grafana/provisioning
+      log:
+        mode: console
+      server:
+        root_url: "{{ control_plane_ingress_endpoint }}/grafana"
+      auth:
+        disable_login_form: true
```

Comment:
That's worth documenting in the changelog, no?

Reply:
To me no, it does not change anything compared to the current behavior since anyway when we use dex we were setting `oauth_auto_login` so we never reach the grafana login form